<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ricardo-PC" name="Default test(failed)" tests="7" failures="2" timestamp="7 Apr 2014 04:23:12 GMT" time="3.081" errors="0">
  <testcase name="usernameShouldNotBeNull" time="2.758" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest"/>
  <testcase name="usernameShouldBeUnique" time="0.12" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest">
    <failure type="org.testng.TestException" message="
Expected exception javax.persistence.RollbackException but got org.testng.TestException: 
Method UserDaoTest.usernameShouldBeUnique()[pri:0, instance:co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest@48d3aac2] should have thrown an exception of class javax.persistence.RollbackException">
      <![CDATA[org.testng.TestException: 
Method UserDaoTest.usernameShouldBeUnique()[pri:0, instance:co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest@48d3aac2] should have thrown an exception of class javax.persistence.RollbackException
Method UserDaoTest.usernameShouldBeUnique()[pri:0, instance:co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest@48d3aac2] should have thrown an exception of class javax.persistence.RollbackException
	... 16 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- usernameShouldBeUnique -->
  <testcase name="saveWithRequiredFields" time="0.099" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest"/>
  <testcase name="saveWithAllFields" time="0.059" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest">
    <failure type="javax.persistence.RollbackException" message="Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &amp;apos;ricardorlg&amp;apos; for key &amp;apos;USERNAME&amp;apos;
Error Code: 1062
Call: INSERT INTO eveun_user (ID_USER, EMAIL, LASTNAME, NAME, PASSWORD, PHONE, ROLE, USERNAME) VALUES (?, ?, ?, ?, ?, ?, ?, ?)
	bind =&amp;gt; [4, ricardo@prueba.com, larrahondo, ricardo, mayhem123, 1234567, GENERAL, ricardorlg]
Query: InsertObjectQuery(co.edu.unal.software.arquitectura.evnetos.server.entities.EveunUser@1c3cd230)">
      <![CDATA[javax.persistence.RollbackException: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'ricardorlg' for key 'USERNAME'
Error Code: 1062
Call: INSERT INTO eveun_user (ID_USER, EMAIL, LASTNAME, NAME, PASSWORD, PHONE, ROLE, USERNAME) VALUES (?, ?, ?, ?, ?, ?, ?, ?)
	bind => [4, ricardo@prueba.com, larrahondo, ricardo, mayhem123, 1234567, GENERAL, ricardorlg]
Query: InsertObjectQuery(co.edu.unal.software.arquitectura.evnetos.server.entities.EveunUser@1c3cd230)
	at org.eclipse.persistence.internal.jpa.transaction.EntityTransactionImpl.commit(EntityTransactionImpl.java:157)
	at com.google.inject.persist.jpa.JpaLocalTxnInterceptor.invoke(JpaLocalTxnInterceptor.java:87)
	at co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest.saveWithAllFields(UserDaoTest.java:63)
Caused by: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.5.1.v20130918-f2b9fc5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'ricardorlg' for key 'USERNAME'
Error Code: 1062
Call: INSERT INTO eveun_user (ID_USER, EMAIL, LASTNAME, NAME, PASSWORD, PHONE, ROLE, USERNAME) VALUES (?, ?, ?, ?, ?, ?, ?, ?)
	bind => [4, ricardo@prueba.com, larrahondo, ricardo, mayhem123, 1234567, GENERAL, ricardorlg]
Query: InsertObjectQuery(co.edu.unal.software.arquitectura.evnetos.server.entities.EveunUser@1c3cd230)
	at org.eclipse.persistence.exceptions.DatabaseException.sqlException(DatabaseException.java:331)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeDirectNoSelect(DatabaseAccessor.java:900)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeNoSelect(DatabaseAccessor.java:962)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.basicExecuteCall(DatabaseAccessor.java:631)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeCall(DatabaseAccessor.java:558)
	at org.eclipse.persistence.internal.sessions.AbstractSession.basicExecuteCall(AbstractSession.java:1991)
	at org.eclipse.persistence.sessions.server.ClientSession.executeCall(ClientSession.java:298)
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:242)
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.executeCall(DatasourceCallQueryMechanism.java:228)
	at org.eclipse.persistence.internal.queries.DatasourceCallQueryMechanism.insertObject(DatasourceCallQueryMechanism.java:377)
	at org.eclipse.persistence.internal.queries.StatementQueryMechanism.insertObject(StatementQueryMechanism.java:165)
	at org.eclipse.persistence.internal.queries.StatementQueryMechanism.insertObject(StatementQueryMechanism.java:180)
	at org.eclipse.persistence.internal.queries.DatabaseQueryMechanism.insertObjectForWrite(DatabaseQueryMechanism.java:489)
	at org.eclipse.persistence.queries.InsertObjectQuery.executeCommit(InsertObjectQuery.java:80)
	at org.eclipse.persistence.queries.InsertObjectQuery.executeCommitWithChangeSet(InsertObjectQuery.java:90)
	at org.eclipse.persistence.internal.queries.DatabaseQueryMechanism.executeWriteWithChangeSet(DatabaseQueryMechanism.java:301)
	at org.eclipse.persistence.queries.WriteObjectQuery.executeDatabaseQuery(WriteObjectQuery.java:58)
	at org.eclipse.persistence.queries.DatabaseQuery.execute(DatabaseQuery.java:899)
	at org.eclipse.persistence.queries.DatabaseQuery.executeInUnitOfWork(DatabaseQuery.java:798)
	at org.eclipse.persistence.queries.ObjectLevelModifyQuery.executeInUnitOfWorkObjectLevelModifyQuery(ObjectLevelModifyQuery.java:108)
	at org.eclipse.persistence.queries.ObjectLevelModifyQuery.executeInUnitOfWork(ObjectLevelModifyQuery.java:85)
	at org.eclipse.persistence.internal.sessions.UnitOfWorkImpl.internalExecuteQuery(UnitOfWorkImpl.java:2896)
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1793)
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1775)
	at org.eclipse.persistence.internal.sessions.AbstractSession.executeQuery(AbstractSession.java:1726)
	at org.eclipse.persistence.internal.sessions.CommitManager.commitNewObjectsForClassWithChangeSet(CommitManager.java:226)
	at org.eclipse.persistence.internal.sessions.CommitManager.commitAllObjectsWithChangeSet(CommitManager.java:125)
	at org.eclipse.persistence.internal.sessions.AbstractSession.writeAllObjectsWithChangeSet(AbstractSession.java:4196)
	at org.eclipse.persistence.internal.sessions.UnitOfWorkImpl.commitToDatabase(UnitOfWorkImpl.java:1441)
	at org.eclipse.persistence.internal.sessions.UnitOfWorkImpl.commitToDatabaseWithChangeSet(UnitOfWorkImpl.java:1531)
	at org.eclipse.persistence.internal.sessions.RepeatableWriteUnitOfWork.commitRootUnitOfWork(RepeatableWriteUnitOfWork.java:277)
	at org.eclipse.persistence.internal.sessions.UnitOfWorkImpl.commitAndResume(UnitOfWorkImpl.java:1169)
	at org.eclipse.persistence.internal.jpa.transaction.EntityTransactionImpl.commit(EntityTransactionImpl.java:132)
	... 26 more
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry 'ricardorlg' for key 'USERNAME'
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	at com.mysql.jdbc.Util.getInstance(Util.java:386)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1041)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4237)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:4169)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2617)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2778)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2834)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2156)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2441)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2366)
	at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2350)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.executeDirectNoSelect(DatabaseAccessor.java:890)
	... 57 more
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- saveWithAllFields -->
  <testcase name="readint" time="0.001" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest">
    <skipped/>
  </testcase> <!-- readint -->
  <testcase name="readString" time="0.001" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest">
    <skipped/>
  </testcase> <!-- readString -->
  <testcase name="delete" time="0.001" classname="co.edu.unal.software.arquitectura.evnetos.server.services.dao.UserDaoTest">
    <skipped/>
  </testcase> <!-- delete -->
</testsuite> <!-- Default test(failed) -->
